airbyte-integrations/connectors/source-jdbc/src/test/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSourceAcceptanceTest.java

```diff
@@ -75,6 +81,14 @@ public boolean supportsSchemas() {
     return true;
   }
 
+  private void assertCustomParametersDontOverwriteDefaultParameters(final Map<String, String> customParameters, final Map<String, String> defaultParameters) {
```

Comment:
Is this identical to the method in AbstractJdbcSource? Why does it need to be copied here (rather than e.g. making the AbstractJdbcSource method `public static`?

Reply:
That's a good point, I was mostly keeping the same access modifiers but this is definitively determine for regression. I've instead have the `AbstractJdbcSource#assertCustomParametersDontOverwriteDefaultParameters` set to `protected static` to limit access because this method seems like something that not all packages need access to